SpatialGDK/Documentation/content/tutorials/offloading-tutorial/offloading-intro.md

```diff
@@ -21,5 +21,4 @@ In this tutorial, you will go through the steps to enable offloading.
 _2019-07-30 Page added as limited editorial review_
 <br/>
 <br/>
-[//]: # (TODO: https://improbableio.atlassian.net/browse/DOC-1142)
-------------
\ No newline at end of file
+------------
```

Comment:
We need to keep this: [//]: # (TODO: https://improbableio.atlassian.net/browse/DOC-1142)

This is a reminder to tech writers, but not visible to users.